Integrations/integration-PacketMail.yml

```diff
@@ -0,0 +1,118 @@
+description: Intel look up for IPS
+configuration:
+- display: API Key
+  name: apikey
+  defaultvalue: ""
+  type: 4
+  required: false
+- display: Use Proxy
+  name: proxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: CSV list of lists that should be considered malicious
+  name: maliciousList
+  defaultvalue: cleanmx_virus,et_c2
+  type: 0
+  required: false
+script:
+  script: |+
+    var initialurl = 'https://www.packetmail.net/iprep.php/';
+    var malLists = params.maliciousList.split(',');
+    for (i = 0 ; i < malLists.length; i++) {
+        malLists[i] = malLists[i].trim();
+    }
+
+    var sendRequest = function(url, param) {
+
+        // handle '/' at the end of the url
+        if (url[url.length - 1] === '/') {
+            url = url.substring(0, url.length - 1);
+        }
+
+        var res = http(
+            url,
+            {
+                Method: 'GET', // Can be POST, PUT, DELETE, HEAD, OPTIONS or CONNECT
+                Headers: {
+                    Accept: ['application/json']
+                }
+            },
+            true,
+            params.proxy
+        );
+
+        if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
+        }
+
+        return JSON.parse(res.Body);
+    };
+
+    // The command input arg holds the command sent from the user.
+    switch (command) {
+        // This is the call made when pressing the integration test button.
+        case 'test-module':
+            res = sendRequest(initialurl + '8.8.8.8' + '?apikey='+params.apikey);
+            if (res.query_result === 'Success') {
+                return 'ok';
+            } else {
+                return res;
+            }
+            break;
+        case 'packetmail-ip':
+            var res =  sendRequest(initialurl + args.ip + '?apikey='+params.apikey);
+            md = '';
+            maliciousLists = '';
+            var keys = Object.keys(res);
+            keys = keys.sort();
+            for (i = 0 ; i < keys.length; i++) {
+                if (md.length > 0) {
+                    md += '\n\n';
+                }
+                if (res[keys[i]] instanceof Object) {
+                    if (malLists.indexOf(keys[i]) >= 0) {
```

Comment:
I don't understand this. You check it in the response and only if in the param mal list you mark it as malicious? if it's planned thats fine but this is a strange behaviour